examples/conversation_with_agent_with_finetuned_model/conversation_with_agent_with_finetuned_model.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+"""
+This script sets up a conversational agent using
+AgentScope with a Hugging Face model.
+It includes initializing a Finetune_DialogAgent,
+loading and fine-tuning a pre-trained model,
+and conducting a dialogue via a sequential pipeline.
+The conversation continues until the user exits.
+Features include model and tokenizer loading,
+and fine-tuning on the databricks-dolly-15k dataset with adjustable parameters.
+"""
+from finetune_dialogagent import Finetune_DialogAgent
+
+# pylint: disable=unused-import
+from huggingface_model import HuggingFaceWrapper
+import agentscope
+from agentscope.agents.user_agent import UserAgent
+from agentscope.pipelines.functional import sequentialpipeline
+
+
+def main() -> None:
+    """A basic conversation demo with a custom model"""
+
+    # Initialize AgentScope with your custom model configuration
+
+    agentscope.init(
+        model_configs=[
+            {
+                "model_type": "huggingface",
+                "config_name": "my_custom_model",
+                # Or another generative model of your choice.
+                # Needed from loading from Hugging Face.
+                "model_id": "openlm-research/open_llama_3b_v2",
```

Comment:
How about using the argument `model_name_or_path` like what transformers library does in `from_pretrained` function rather than providing two arguments?

Reply:
Changed according to suggestion.